contracts/FeralfileArtworkV4.sol

```diff
@@ -272,7 +262,7 @@ contract FeralfileExhibitionV4 is
             // distribute royalty
             for (uint256 j = 0; j < saleData_.royalties[i].length; j++) {
                 payable(saleData_.royalties[i][j].recipient).transfer(
-                    (itemCost * saleData_.royalties[i][j].bps) / 10000
+                    (itemRevenue * saleData_.royalties[i][j].bps) / 10000
```

Comment:
just aware that we should use something like `revenueShares` instead of `royalties`.